pkg/justification/processor.go

```diff
@@ -98,7 +101,12 @@ func (p *Processor) CreateToken(ctx context.Context, request *jvspb.CreateJustif
 }
 
 func (p *Processor) getLatestSigner(ctx context.Context) (*signerWithID, error) {
-	ver, err := jvscrypto.GetLatestKeyVersion(ctx, p.kms, p.config.KeyName)
+	var jvsKeyConfig config.JVSKeyConfig
+	err := p.remoteConfig.Unmarshal(ctx, &jvsKeyConfig)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get remoteConfig: %w", err)
+	}
```

Comment:
```suggestion
	if err := p.remoteConfig.Unmarshal(ctx, &jvsKeyConfig); err != nil {
		return nil, fmt.Errorf("failed to get remoteConfig: %w", err)
	}
```

Reply:
nit: Since now we have Get func, would it be easier to use that instead?

Reply:
done